swipe.js

```diff
@@ -48,7 +48,7 @@
     var slides, slidePos, width, length;
     options = options || {};
     var index = parseInt(options.startSlide, 10) || 0;
-    var speed = options.speed || 300;
+    var speed = options.speed === undefined ? 300 : options.speed;
```

Comment:
direct `undefined` comparison is not a good practice. I think such check should be enough in this case:
```js
var speed = !isNaN(options.speed) ? Number(options.speed) : 300;
```